Lib/calendar.py

```diff
@@ -139,6 +139,24 @@ def __len__(self):
 month_name = _localized_month('%B')
 month_abbr = _localized_month('%b')
 
+# Check if the platform supports the %OB format code
+def _is_alt_mon_available():
+    try:
+        datetime.date(2001, 1, 1).strftime('%OB')
+        datetime.date(2001, 1, 1).strftime('%Ob')
+    except ValueError:
+        return False
+    return True
+
+# In Greek and in many Slavic and Baltic languages, "%OB" will produce
+# the month in nominative case.
+if _is_alt_mon_available():
+    alt_month_name = _localized_month('%OB')
+    alt_month_abbr = _localized_month('%Ob')
```

Comment:
These will probably need a documentation entry in `calendar.rst` as well (and maybe update `month_name`/`month_abbr` to make it clear that they return the genitive case?)